scst-scan/scan-2-0.hbs.md

```diff
@@ -1,4 +1,4 @@
-# Supply Chain Security Tools - Scan 2.0
+# Supply Chain Security Tools - Scan 2.0 with Supply Chain Choreographer for Tanzu
```

Comment:
```suggestion
# Supply Chain Security Tools - Scan 2.0
```
Let's not complicate this.  Keep it simple.  When Supply Chains Goes GA we'll try to time that so we depreciate scan 1.0 and can "hide" those docs, and make Supply Chains more prominent.